Plugin/src/main/java/com/gmail/filoghost/holographicdisplays/placeholder/Placeholder.java

```diff
@@ -1,102 +1,102 @@
-/*

```

Comment:
Fix diff

Reply:
i really dont know whats apening on github because on local i dont have that miss :(